src/microns_to_nwb/convert_microns_to_nwb.py

```diff
@@ -0,0 +1,16 @@
+from tools.behavior import add_eye_tracking, add_treadmill
+from tools.intervals import add_trials
+from tools.nwb_helpers import start_nwb
+from tools.ophys import add_ophys
+from tools.stimulus import add_stimulus
+
+
+def build_nwb(scan_key):
+    nwb = start_nwb(scan_key)
+    add_stimulus(scan_key, nwb)
+    add_eye_tracking(scan_key, nwb)
+    add_treadmill(scan_key, nwb)
+    add_trials(scan_key, nwb)
+    add_ophys(scan_key, nwb)
```

Comment:
Are all of these fields necessarily available across sessions? Or is there variability across sessions where maybe one of them is missing? If that's possible, I'd build in some safe checks to these functions like early returns in case of non-availability

Reply:
let's see if they are ever missing first